app/components/preNoted/blockedPopper.tsx

```diff
@@ -0,0 +1,64 @@
+import * as React from "react";
+import Popper from "@material-ui/core/Popper";
+import { withStyles } from "../../helpers/withStylesHelper";
+import { getBlockedBubbleContext } from "./constants";
+import { blockUnverifiedUser, AUTH_LEVEL } from "../../helpers/checkAuthDialog";
+import { BUBBLE_CONTEXT_TYPE } from "../../helpers/getBubbleContextType";
+const store = require("store");
+const styles = require("./blockedPopper.scss");
+
+interface BlockedPopperProps {
+  isOpen: boolean;
+  handleOnClickAwayFunc: () => void;
+  anchorEl: HTMLDivElement | null;
+  buttonClickAction: Scinapse.ActionTicket.ActionTagType;
+}
+
+const BlockedPopperContent: React.FC<{
+  buttonClickAction: Scinapse.ActionTicket.ActionTagType;
+  onClickAwayButton: () => void;
+}> = props => {
+  const { buttonClickAction, onClickAwayButton } = props;
+  const bubbleContext = getBlockedBubbleContext(String(store.get(BUBBLE_CONTEXT_TYPE)), buttonClickAction);
+
+  return (
+    <div className={styles.bubbleContent}>
+      <div className={styles.titleContext}>{bubbleContext.title}</div>
+      <div className={styles.mainContext}>{bubbleContext.mainText}</div>
+      <button
+        className={styles.joinBtn}
+        onClick={e => {
+          e.preventDefault();
+          blockUnverifiedUser({
```

Comment:
missing async await logic.

Reply:
is it right the QA is passed?
before asking QA or PR please test all features by yourself.

i guess this function cannot block unverified user.

Reply:
I have found that it works very well when I tested the relevant logic.

However, it is a good idea to write the code clearly, so I will add that logic.